spec/Section 7 -- Response.md

```diff
@@ -10,31 +10,74 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A response to a GraphQL request must be a map.
+The result of a GraphQL request must be either a single initial response or an
```

Comment:
A GraphQL request returns either a _response_, a _response stream_, or an _incremental stream_.

Reply:
I think I've mentioned this before, but returning an object like this feels dangerous because it makes it more probable that the stream will not be handled? IMO GraphQL should either return an object OR a stream, and never an object that has a stream inside it.